### PR TITLE
Mention Elastic Agent monitoring in the Fleet docs

### LINKED ISSUE #770: Discuss Elastic Agent monitoring
The Elastic Agent integration is now installed by default. We need to mention this in the quick start and in other places where we discuss integrations that are installed by default (System integration).

We probably also need a topic about monitoring Elastic Agents so users understand that it's on by default and know where to look for metrics and agent logs.

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #770.

Most of the info requested in #770 is already covered in #897.

The docs build for this PR will fail until https://github.com/elastic/observability-docs/pull/897 is merged.

### CODE REVIEW

File: docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc [hunk 1]
```diff
@@ -88,7 +88,13 @@ automatically if the system is rebooted.
 To confirm that {agent} is installed and running, go to the **Agents** tab in
 {fleet}. 
 
-If you run into problems, see <<fleet-troubleshooting>>.
+If you run into problems:
+
+* Check the {agent} logs. If you use the default policy, agent logs and metrics
+are collected automatically unless you change the default settings. For more
+information,refer to <<elastic-agent-logging>>.
```
Comment (bmorelli25) [inline docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc:95]:
```suggestion
information, refer to <<elastic-agent-logging>>.
```

File: docs/en/ingest-management/getting-started-traces.asciidoc [hunk 2]
```diff
@@ -103,8 +103,13 @@ image::images/kibana-fleet-agents.png[{fleet} showing enrolled agents]
 TIP: If the status hangs at Enrolling, make sure the `elastic-agent` process
 is running.
 
-If you run into problems, see <<fleet-troubleshooting>>.
+If you run into problems:
 
+* Check the {agent} logs. If you use the default policy, agent logs and metrics
+are collected automatically unless you change the default settings. For more
+information,refer to <<elastic-agent-logging>>.
```
Comment (bmorelli25) [inline docs/en/ingest-management/getting-started-traces.asciidoc:110]:
```suggestion
information, refer to <<elastic-agent-logging>>.
```

File: docs/en/ingest-management/getting-started.asciidoc [hunk 3]
```diff
@@ -95,8 +95,15 @@ image::images/kibana-fleet-agents.png[{fleet} showing enrolled agents]
 TIP: If the status hangs at Enrolling, make sure the `elastic-agent` process
 is running.
 
-If you run into problems, see <<fleet-troubleshooting>>.
+If you run into problems:
+
+* Check the {agent} logs. If you use the default policy, agent logs and metrics
+are collected automatically unless you change the default settings. For more
+information,refer to <<elastic-agent-logging>>.
```
Comment (bmorelli25) [inline docs/en/ingest-management/getting-started.asciidoc:102]:
```suggestion
information, refer to <<elastic-agent-logging>>.
```

File: docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc [hunk 4]
```diff
@@ -88,7 +88,13 @@ automatically if the system is rebooted.
 To confirm that {agent} is installed and running, go to the **Agents** tab in
 {fleet}. 
 
-If you run into problems, see <<fleet-troubleshooting>>.
+If you run into problems:
+
+* Check the {agent} logs. If you use the default policy, agent logs and metrics
```
Comment (bmorelli25) [inline docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc:93]: Is lowercase `agent` correct here?

**Review by bmorelli25 — APPROVED** [review]

File: docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc (hunk 4 above)
Comment (bmorelli25) [reply docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc:93]: Nevermind. I've answered my own question after reading through some of your other PRs.

**Comment by dedemorton** [comment]
@elasticmachine run elasticsearch-ci/docs

**Pull request merged**